docs/misc/examples.md

```diff
@@ -26,6 +26,54 @@ services:
 
 [Source](https://github.com/itzg/docker-minecraft-server/blob/master/examples/geyser/docker-compose.yml)
 
+## Lazymc - Put your Minecraft server to rest when idle
+
+With [lazymc-docker-proxy](https://github.com/joesturge/lazymc-docker-proxy/settings) you are able to use [lazymc](https://github.com/timvisee/lazymc) with the minecraft container.
+
+```yaml
+services:
+    lazymc:
+        container_name: lazymc
+        image: ghcr.io/joesturge/lazymc-docker-proxy:latest
+        environment:
+            # Point to the service name of the Minecraft server
+            - SERVER_ADDRESS=mc:25565
+            # Required to find the container to manage it
+            - LAZYMC_GROUP=mc

```

Comment:
For consistency can you switch this to object syntax:

```suggestion
            # Point to the service name of the Minecraft server
            SERVER_ADDRESS: mc:25565
            # Required to find the container to manage it
            LAZYMC_GROUP: mc
```

Reply:
fixed inconsistancies